### ISSUE TITLE
Technologies and Projects relationship

### ISSUE THREAD

**Issue by rafaelcorreiapoli** [envisioning/tdb#306]
@arthursoares 
A relação entre uma Technology e um Project está descrita pela technologiesStash do Project?

**Comment by arthursoares** [comment]
Sim! Deveria ser a associação pelo Stash. Não sei como vocês farão na questão de DB, achei que armazenariam nos dois objetos a relação por dinâmica de query.
